project-handbook/reuse.md

```diff
@@ -0,0 +1,41 @@
+# Adopt REUSE to ensure licensing transparency
+
+OpenRail projects are supposed to be safely used in a compliant way. This is why from Incubation Stage 2 onwards, projects shall adopt the [REUSE](https://reuse.software/) best practices for licensing and copyright information.
+
+The fundamental idea is that for all files within a project there is unambiguous information about their license and copyright. REUSE consists of a standard and various tools and services that help with reaching and ensuring this clarity.
+
+This document helps with the adoption by providing links to relevant material and making recommendations for OpenRail projects.
+
+## Resources for understanding REUSE
+
+Instead of duplicating guides, we refer to helpful material that already exists and is maintained by REUSE:
+
+* [Tutorial](https://reuse.software/tutorial/) on how to become REUSE compliant: make a repository REUSE compliant in simple steps, without delving into details and edge cases.
+* [FAQ](https://reuse.software/faq/) on general questions but also how to deal with edgier cases in your project.
+* [Tools and scripts](https://reuse.software/dev/) specifically for developers, such as the helper tool, API, pre-commit hooks, and CI checks
+
+## Recommended process for OpenRail projects
+
+For Incubation Stage 2, we require projects to be fully REUSE compliant. This is measured by whether the `reuse lint` command exits successfully, but requires that the projects carefully check the licenses and copyright of all their files.
+
+### Analyse your codebase
+
+When thinking about licensing and copyright of all your files, tricky questions may arise:
+
+* Are all files licensed the same way, or have files been copied from other projects under different licenses and copyrights?
+* Who owns the copyright of all the files? Just one organization, or were there contractors or external companies who acted under their own copyright?
+* Are there files that are not licensed under the "main" license, e.g., icons, fonts, graphics, pictures?
+* Are there trademarks or additional restrictions on parts of the projects, e.g. logos?
```

Comment:
What is the answer REUSE gives for these trademark questions? Is there a standardized way or a best practice how to capture trademark information?

Reply:
For trademarks, there isn't a defined mechanism with REUSE, as the scope is only licenses and copyright. You could add any SPDX file information as well, but to my knowledge, there is no defined field for trademarks.

If a project concludes in this step that e.g. a logo can be licensed as Open Source but is covered by a trademark, we would have to discuss how to communicate that (probably in the README).

If such a logo cannot be licensed as Open Source, REUSE allows a custom license: https://reuse.software/faq/#custom-license

Reply:
Maybe it would be better to leave out the trademark question in this block as it might be misleading in terms of what can be covered directly by REUSE. Instead we could maybe add a note at the end of the document that REUSE covers copyright and trademark considerations should be communicated via the README.

Reply:
OK, tried that in 7bc91b9c042f5fcbdd7a1f23e2ebde815cce4760

I am still not very happy about it, especially as we actually want that all trademarks are moved to OpenRail in the first place. But I think it's good enough and yet another pointer to projects to check if there is a legal minefield.

Reply:
We can mention that as well, that trademarks of OpenRail projects belong to the OpenRail Association. But also in this case we would want to be transparent about trademark restrictions. So mentioning that in the README is the right thing in any case.